app/src/androidTest/java/org/projectbuendia/client/ui/chart/PatientChartActivityTest.java

```diff
@@ -148,13 +158,15 @@ public void testPatientChart_CanOpenEncounterFormMultipleTimes() {
     }*/
 
     protected void openEncounterForm() {
-        // TODO: this method is still producing intermittent errors
+        // Wait until the overflow menu button is available.
+        expectVisibleSoon(Espresso.onView(OVERFLOW_BUTTON_MATCHER));
         openActionBarOptionsMenu();
 
         EventBusIdlingResource<FetchXformSucceededEvent> xformIdlingResource =
-            new EventBusIdlingResource<FetchXformSucceededEvent>(
-                UUID.randomUUID().toString(), mEventBus);
-        click(viewWithText("[test] Form"));
+                new EventBusIdlingResource<>(UUID.randomUUID().toString(), mEventBus);
+        ViewInteraction testForm = viewWithText("[test] Form");
+        expectVisibleSoon(testForm);
+        click(testForm);
         Espresso.registerIdlingResources(xformIdlingResource);
```

Comment:
The above lines are great! I tried to fix the problem of "[test] Form" not found but my limited knowledge of android / spresso stopped me from doing it.
I just tested it and it did not happen once, so I summon your mighty force to repeat this with the webview idling resource. :) (of course, when you have time). It is another thing I have an branch but did not have success implementing.